devtools/jenkins/jenkins.sh

```diff
@@ -6,13 +6,22 @@ source activate $envname
 python setup.py install
 
 rm examples/results.csv || true
+export DEEPCHEM_DATA_DIR='/tmp/molnet_'$envname
+mkdir $DEEPCHEM_DATA_DIR
 cd examples
-python benchmark.py -d tox21
-export retval1=$?
+python benchmark.py -d tox21 -m weave -m graphconv -m tf_robust -m tf -m irv -m xgb -m logreg --seed 123
```

Comment:
How long do the benchmarks take to run in total?

Reply:
4 to 5 hours at this setting